src/Benchmarks/Benchmarks.csproj

```diff
@@ -1,34 +1,33 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <Import Project="..\..\build\common.props" />
-
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
This file previously used only properties defined in dependencies.props. So, if what you wanted was that (and I misread the issue completely), #251 was fixed before you filed it.